### DISCUSSION TITLE
IE 11 compatibility

### DISCUSSION THREAD

**Discussion by vahidsamimi** [codex-team/editor.js#1909] [Q&A]
Hi all, 

the first question is editor.js with IE 11 compatible?
and its possible to import rtf value in editor?

as example I have this rft value and I want to show into editor

 var rtfStr = "{\rtf1\ansi\ansicpg1252\deff0\nouicompat\deflang1033{\fonttbl{\f0\fnil\fcharset0 Segoe UI;}} {\*\generator Riched20 10.0.17763}\paperw11906\paperh16838\margl1134\margr1134\margt1134\margb1134\viewkind4\uc1 \pard\f0\fs36 test \fs20\par}"

**Comment by AndreaTosti** [comment]
I don't know how to answer you on internet explorer 11 but for **similar** needs I build ES5 with the following lines (inside `package.json`):

```
  "scripts": {
    "build": "rimraf dist/* && yarn svg && yarn build:prod && babel dist -d dist && uglifyjs dist/editor.js -o dist/editor.js",
....
},
....
  "devDependencies": {
....
    "@babel/cli": "^7.10.5",
    "uglify-js": "^3.10.1",
....
}
